scrapinghub/client/__init__.py

```diff
@@ -0,0 +1,91 @@
+from scrapinghub import Connection as _Connection
+from scrapinghub import HubstorageClient as _HubstorageClient
+
+from .projects import Projects
+from .exceptions import wrap_http_errors
+
+from .utils import parse_auth
+from .utils import parse_project_id, parse_job_key
+
+
+class Connection(_Connection):
+
+    @wrap_http_errors
+    def _request(self, *args, **kwargs):
+        return super(Connection, self)._request(*args, **kwargs)
+
+
+class HubstorageClient(_HubstorageClient):
+
+    @wrap_http_errors
+    def request(self, *args, **kwargs):
+        return super(HubstorageClient, self).request(*args, **kwargs)
```

Comment:
let's add `__all__ = ['ScrapinghubClient']` - to exclude possible clashes with HS client and Connection on import